src/coreclr/jit/compiler.h

```diff
@@ -4867,6 +4867,10 @@ class Compiler
     void fgPerNodeLocalVarLiveness(GenTree* node);
     void fgPerBlockLocalVarLiveness();
 
+#if defined(FEATURE_HW_INTRINSICS)
+    void fgPerNodeLocalVarLiveness(GenTreeHWIntrinsic* hwintrinsic);
```

Comment:
Nit: can we avoid overloading `fgPerNodeLocalVarLiveness` here and call it e. g. `fgPerHWIntrinsicNodeLocalVarLiveness`?

Overloading like this makes a it a little bit harder to work code in debugger, e. g. it will make the stack trace contain two distinct methods with the same name.

Reply:
Same reasoning given below. Overload resolution is a powerful and natural tool of the C++ language. Debuggers that are working with C++ need to (and most often do) support such functionality, much as is required to work with C#/.NET code.

If there is a subset of tools, such as WinDBG, where this is not "well supported" then we should push on them to improve their tooling instead.